python/ctranslate2/extensions.py

```diff
@@ -332,16 +343,132 @@ def generator_generate_tokens(
     )
 
 
+async def generator_async_generate_tokens(
+    generator: Generator,
+    start_tokens: Union[List[str], List[List[str]]],
+    max_batch_size: int = 32,
```

Comment:
Same comment about not rebatching the input by default.

```suggestion
    max_batch_size: int = 0,
```